$NetBSD: patch-CMakeLists.txt,v 1.1 2019/08/27 20:39:06 markd Exp $

Something in FindFFmpeg is caching failures so test lowest
version first.

--- CMakeLists.txt.orig	2019-08-03 19:54:00.000000000 +0000
+++ CMakeLists.txt
@@ -72,9 +72,9 @@ set_package_properties(LibExiv2 PROPERTI
                        TYPE OPTIONAL
                        PURPOSE "Support for image metadata")
 
+find_package(FFmpeg 55.27 COMPONENTS AVUTIL)
 find_package(FFmpeg 57.48 COMPONENTS AVCODEC)
 find_package(FFmpeg 57.40 COMPONENTS AVFORMAT)
-find_package(FFmpeg 55.27 COMPONENTS AVUTIL)
 set_package_properties(FFmpeg PROPERTIES DESCRIPTION "Video Tag reader"
                        URL "https://ffmpeg.org/" TYPE OPTIONAL
                        PURPOSE "Support for video metadata")
